pyproject.toml

```diff
@@ -79,6 +79,10 @@ deb = [
     # labgrid[snmp]
     "pysnmp-lextudio>=4.4.12",
 ]
+contrib = [
+    "fastapi>=0.104.1",
+    "uvicorn>=0.24.0",
+]
```

Comment:
I don't think we should add dependencies for contrib scripts in our pyproject.toml. Better document this, either in a module doc string in the script itself or maybe add a README in contrib?

Reply:
Okay, I'll add a requirements file in contrib/ with a readme.

A colleague also mentioned that this could be extended to serve as a web interface for users that aren't too familiar with command line interfaces. I might rename the script so that other routes can be added later.